pulseapi/users/admin.py

```diff
@@ -6,21 +6,32 @@
 from django.utils.html import format_html
 
 from .models import EmailUser
+from pulseapi.entries.models import Entry
 from pulseapi.profiles.models import UserProfile, UserBookmarks
+from pulseapi.utility.get_admin_url import get_admin_url
+
+class EntryInline(admin.TabularInline):
+    model = Entry
+    verbose_name = 'entries'
```

Comment:
I though that for inlines (which don't exist in the admin nav panel) those were the same, but maybe they're not.